nasa/openmct
# [Display Layout] Exclude layout objects from 'missing' object check

## Overview
Addresses #3262

## Testing Instructions
1. create a display layout
1. create layout objects
1. Click them and observe that console errors are not thrown for status being undefined

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N/A |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |
| Testing instructions included? | Y |
